Dockerfile

```diff
@@ -1,34 +1,44 @@
 FROM ubuntu:18.04
 
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
```suggestion
ENV DEBIAN_FRONTEND=noninteractive
```

Reply:
I had made it an `ARG` so that it does not get passed into the final container when it's run. I guess it wouldn't hurt to have it as an env var in the running container, but is there any reason this needs to be an `ENV`?

Reply:
Sorry! You are right!!!
While investigating further, it seems that it is even discouraged, maybe we should drop it altogether?
https://docs.docker.com/engine/faq/#why-is-debian_frontendnoninteractive-discouraged-in-dockerfiles#why-is-debian_frontendnoninteractive-discouraged-in-dockerfiles

Reply:
That's ok!
It looks like it is mainly discouraged so that containers that build from this image don't have weird package manager issues because they never get dialog popups. That's part of the reason I added it as an `ARG`. An `ARG` won't show up after the initial Docker build so that it doesn't get inherited if someone builds from this image.
I agree, though, that it could probably be removed.

Reply:
Actually, disabling it causes `tzdata` to ask which region to use interactively. I will leave it there as an `ARG` for now.

Reply:
Yep! You are right again! Thanks a lot!